### ISSUE TITLE
Error about divergences when sampling with iter_sampling = 0 and save_warmup = 1

### ISSUE THREAD

**Issue by bbbales2** [stan-dev/cmdstanr#286]
**Describe the bug**

There's an error about divergences when running chains with no sampling and only warmup and saving the warmup.

**To Reproduce**

Do something like this:

```fit = model$sample(data = data, save_warmup = 1, iter_warmup = 10, iter_sampling = 0)```

You'll see this error in the console:
```
All 4 chains finished successfully.
Mean chain execution time: 0.1 seconds.
Total execution time: 0.5 seconds.
Error in if (num_of_divergences > 0) { : 
  missing value where TRUE/FALSE needed
```

**Expected behavior**

There should be no error

**Operating system**

Ubuntu 18.04

**CmdStanR version number**

0.1.1

**Comment by rok-cesnovar** [comment]
Good catch! num_of_divergences is NULL in this case and that condition does not work for NULL.
